src/main/java/org/mskcc/cbio/portal/scripts/ImportStructuralVariantData.java

```diff
@@ -175,27 +178,34 @@ public void importData() throws IOException, DaoException {
                         // Add structural variant
                         DaoStructuralVariant.addStructuralVariantToBulkLoader(structuralVariant);
 
-                        // Add sample to sample profile list, which is important for gene panels
-                        if (!DaoSampleProfile.sampleExistsInGeneticProfile(sample.getInternalId(), geneticProfileId) && !sampleSet.contains(sample.getStableId())) {
-                            if (genePanel != null) {
-                                DaoSampleProfile.addSampleProfile(sample.getInternalId(), geneticProfileId, GeneticProfileUtil.getGenePanelId(genePanel));
-                            } else {
-                                DaoSampleProfile.addSampleProfile(sample.getInternalId(), geneticProfileId, null);
-                            }
-                        }
-                        sampleSet.add(sample.getStableId());
-                        orderedSampleList.add(sample.getInternalId());
+                        sampleIds.add(sample.getInternalId());
                     }
                 }
             }
         }
-        DaoGeneticProfileSamples.addGeneticProfileSamples(geneticProfileId, orderedSampleList);
+        // TODO the dao methods could receive a set of sample ids (like the deletion does) instead of looping
```

Comment:
will it be done here or is there a ticket?

Reply:
let me do it here

Reply:
here it is https://github.com/cBioPortal/cbioportal-core/pull/47/commits/c4d5eccf742a80cf7c4afcd087999cd2badae5a8